src/Sylius/Bundle/ApiBundle/Command/Admin/RequestResetPasswordEmail.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ *  This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Command\Admin;
+
+use Sylius\Bundle\ApiBundle\Command\IriToIdentifierConversionAwareInterface;
+
+/** @experimental */
+class RequestResetPasswordEmail implements IriToIdentifierConversionAwareInterface
```

Comment:
Why did you use this interface?
```suggestion
class RequestResetPasswordEmail
```